distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -344,6 +345,29 @@ application/x-msdownload,
 application/x-msmetafile
 |Yes
 
+6+^h|Solr Provider Properties
+
+|Solr Client
+|solr.client
+|String
+|Solr Catalog Provider selected. See <<_configuring_thread_pools,Configuring Thread Pools>>
+|HttpSolrClient, CloudSolrClient, EmbeddedSolrServer
+|Default
+
+|Solr Catalog Provider Url
+|solr.http.url
+|String
+|Solr Catalog Provider Url (can be either a Standalone Solr Server or a Cluster of Standalone Solr Servers)
```

Comment:
It is only a cluster of servers when it is in cloud mode..right? When in Http mode it is a standalone server.